README.md

```diff
@@ -77,19 +77,15 @@ Here's a rough sketch of how  I imagine the milestones of this project:
 
 ## Become a contributor
 
-### Contribute to course material
-If you are interested in contributing to course development, please fill the following form: https://danielkantor196881.typeform.com/to/V00Paz. The project is in a very early stage right now, so you might not be able to contribute right away. Your work will be released in this GitHub repository and you will show up as a contributor here.
+We welcome contributions for course material as well as development contributions. In order to know more regarding your contributions to our project please checkout the [contributing guide](docs/CONTRIBUTING_GUIDE.md)
 
-### UI translation
-
-[See docs](https://librelingo.readthedocs.io/en/latest/translation/#translation)
+## CODE OF CONDUCT
 
-### Development and other forms of contribution
+We follow certain guidelines in order to maintain this repository. Before you start contributing we highly urge you to read our guidelines [here](https://librelingo.readthedocs.io/en/latest/CODE_OF_CONDUCT/)
```

Comment:
that link will (I hope) be broken, because CODE_OF_CONDUCT should be in the repo root, and hence will not be part of the dev docs. That's ok, becuase it's its standard place. 

Also, this extra text adds little value I think. I would simply link Code of Conduct, but I'd rather link it from CONTRIBUTING.md, what do you think? If you really want to put some text around it, I think it's enough to say "Please read our Code of Conduct"

Reply:
Yes @kantord I will link the code of conduct from CONTRIBUTING.md doc. Where should I place the code of conduct.md file?